docs/source/advanced/cmake.rst

```diff
@@ -165,7 +165,7 @@ OpenMP 5
 alpaka_ACC_ANY_BT_OMP5_ENABLE
   .. code-block::
 
-     Enable the OpenMP 5.0 CPU block and block thread back-end.
+     Enable the OpenMP 5.0 any target block and block thread back-end.
```

Comment:
```suggestion
     Enable the OpenMP 5.0 back-end (for any target, including offloading support).
```

Reply:
Does "any target" not include offloading targets?

Reply:
True.